textrepo-app/db/V001__clear.sql

```diff
@@ -5,11 +5,10 @@
 
 drop table if exists files_metadata;
 drop table if exists documents_metadata;
-drop index if exists doc_files_by_file_id;
```

Comment:
Als we geen 'drop index' meer doen, verdwijnen oude indexen dan nog wel (wat als ze dezelfde naam hadden, maar straks aangemaakt worden met andere kolommen bijv?)
Kan zijn dat ze automagisch worden weggegooid als de bijbehorende tabel verdwijnt, maar ik dacht dat het belangrijk was dat we alles in 'reverse order' weggooien van hoe het is aangemaakt. Zoals in het commentaar bovenin staat.

Reply:
Zodra je een tabel verwijdert, worden de bijbehorende indexen verwijderd:
_DROP TABLE always removes any indexes, rules, triggers, and constraints that exist for the target table._
https://www.postgresql.org/docs/8.2/sql-droptable.html

Reply:
Postgres ruimt zelf alles netjes op. https://stackoverflow.com/a/38482683